pkg/config/types.go

```diff
@@ -17,13 +17,14 @@ type InsightsConfigurationSerialized struct {
 }
 
 type DataReportingSerialized struct {
-	Interval                    string `json:"interval,omitempty"`
-	UploadEndpoint              string `json:"uploadEndpoint,omitempty"`
-	DownloadEndpoint            string `json:"downloadEndpoint,omitempty"`
-	DownloadEndpointTechPreview string `json:"downloadEndpointTechPreview,omitempty"`
-	StoragePath                 string `json:"storagePath,omitempty"`
-	ConditionalGathererEndpoint string `json:"conditionalGathererEndpoint,omitempty"`
-	ProcessingStatusEndpoint    string `json:"processingStatusEndpoint"`
+	Interval                    string             `json:"interval,omitempty"`
+	UploadEndpoint              string             `json:"uploadEndpoint,omitempty"`
+	DownloadEndpoint            string             `json:"downloadEndpoint,omitempty"`
+	DownloadEndpointTechPreview string             `json:"downloadEndpointTechPreview,omitempty"`
+	StoragePath                 string             `json:"storagePath,omitempty"`
+	ConditionalGathererEndpoint string             `json:"conditionalGathererEndpoint,omitempty"`
+	ProcessingStatusEndpoint    string             `json:"processingStatusEndpoint,omitempty"`
+	Obfuscation                 []ObfuscationValue `json:"obfuscation,omitempty"`
```

Comment:
why using []ObfuscationValue instead of Obfuscation type?

Reply:
Ha... good catch. Thank you